src/cpp/core/system/PosixSystem.cpp

```diff
@@ -1298,7 +1340,7 @@ Error osResourceLimit(ResourceLimit limit, int* pLimit)
       case CpuLimit:
          *pLimit = RLIMIT_CPU;
          break;
-#ifndef __APPLE__
+#if !defined(__APPLE__) && !defined(__FreeBSD__) && !defined(__DragonFly__) && !defined(__OpenBSD__) && !defined(__NetBSD__)
```

Comment:
I wonder if all of these could / should just be an `#ifdef __linux__` or similar?

Reply:
Yes, I'll change it.